save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Execution.kt

```diff
@@ -156,7 +156,7 @@ class Execution(
             status = ExecutionStatus.RUNNING,
             testSuiteIds = null,
             batchSize = 20,
-            type = ExecutionType.GIT,
+            type = TestingType.PUBLIC_TESTS,
```

Comment:
For some reason we had different types in stubs for `Execution` and `ExectuionDto`. Will set both to `PUBLIC`